photon-ml/src/main/scala/com/linkedin/photon/ml/data/GameConverters.scala

```diff
@@ -63,7 +63,7 @@ object GameConverters {
     // Add unique id
     val recordsWithUniqueId = data.withColumn(UNIQUE_ID_COLUMN_NAME, monotonicallyIncreasingId)
 
-    recordsWithUniqueId.map { row: Row =>
+    recordsWithUniqueId.rdd.map { row: Row =>
```

Comment:
Wow, that was fast! Thanks @XianXing 

I decided to do it this way to avoid importing implicits, and also to mirror how they do it internally in spark.ml, e.g.:

https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala#L330